docs/book/src/SUMMARY.md

```diff
@@ -32,6 +32,7 @@
             - [Implementing Topology Mutation Hook Extensions](./tasks/experimental-features/runtime-sdk/implement-topology-mutation-hook.md)
             - [Deploying Runtime Extensions](./tasks/experimental-features/runtime-sdk/deploy-runtime-extension.md)
         - [Ignition Bootstrap configuration](./tasks/experimental-features/ignition.md)
+    - [Migrating to user-managed cert-manager](./tasks/migrating-to-user-managed-cert-manager.md)
```

Comment:
This might be better organised if associated with other clusterctl cert management configuration topics here: https://cluster-api.sigs.k8s.io/clusterctl/configuration.html#cert-manager-configuration